src/main/java/com/tickets/model/dto/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package com.tickets.model.dto;
 
+import com.sun.istack.NotNull;
```

Comment:
import javax.validation.constraints.NotNull;

Reply:
Have you tested your code?

Reply:
@boroda4436 yes. Maybe, I didn't test it correctly